go/teams/keys.go

```diff
@@ -186,7 +186,7 @@ func (t *TeamKeyManager) recipientBoxes(secret []byte, nonce *nonce24, senderKey
 			return nil, err
 		}
 
-		boxes[uid] = base64.StdEncoding.EncodeToString(encodedArray)
+		boxes[uv.Uid] = base64.StdEncoding.EncodeToString(encodedArray)
```

Comment:
Well right after this, we send up to the server with a map key via regular `UID`s. So we'd just have to down-convert it.